### PR TITLE
task/Update release calendar with new schedule

### PR DESCRIPTION
**Pull request by RichardEustace**

### CODE REVIEW

File: docs/release-calendar/index.md [hunk 1]
```diff
@@ -101,7 +101,137 @@ title: Release Calendar
...
+      <td class="govuk-table__cell">Sandbox</td>
+      <td class="govuk-table__cell">Application</td>
+    </tr>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">17/05/2021</td>
+      <td class="govuk-table__cell">Production</td>
+      <td class="govuk-table__cell">Application</td>
+    </tr>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">17/05/2021</td>
```
Comment (cpaul92) [inline docs/release-calendar/index.md:209]: Is this right? The previous production release is the same day? I think this should be `27/05/2021`

File: docs/release-calendar/index.md [hunk 2]
```diff
@@ -101,7 +101,137 @@ title: Release Calendar
...
+      <td class="govuk-table__cell">Sandbox</td>
+      <td class="govuk-table__cell">Application</td>
+    </tr>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">14/06/2021</td>
+      <td class="govuk-table__cell">Production</td>
+      <td class="govuk-table__cell">Application</td>
+    </tr>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">14/06/2021</td>
```
Comment (cpaul92) [inline docs/release-calendar/index.md:229]: Same question here? Should this be `24/06/2021`?

Comment (RichardEustace) [reply docs/release-calendar/index.md]: Yep 24th - Good spot

File: docs/release-calendar/index.md (hunk 1 above)
Comment (RichardEustace) [reply docs/release-calendar/index.md]: Yep 27th Good spot

**Review by cpaul92 — APPROVED** [review]

**Review by pwright08 — APPROVED** [review]

**Pull request merged**